execution_workflows/DaPars/bin/check_bedgraph.py

```diff
@@ -21,9 +21,9 @@ def check_bedgraph(file_in):
 	fin = open(file_in, "rt")
 
 	for line in fin:
-		if line[0] != '#' and line[:3] != "chr":
-			msg = "Found a row in the sample file " + file_in + " without leading 'chr' in the chromosome column.\n" + \
-				" Please use a file with leading 'chr' for all sequence regions."
+		if line[0] != '#' and '.' not in line and line[:3] != "chr":
```

Comment:
As I understand it, if no dot is in the line, nor a comment (#) and it does not start with "chr", it is not a valid format.
I don't think the dot check will help, as it can be anywhere in the line. In the example you give, the column "rep" contains dots in the decimals for example. 
Maybe you need to check whether the line starts with `(1-22,X,Y,M)`, as you describe in the message.

Reply:
This is a good point. Maybe another, more general, fix is to check that there are _any_ lines in the bedgraph that lead with "chr", otherwise throw the error message. In theory, someone may use this execution workflow for non-human organisms so checking for the specific chromosome nomenclature we see in human (or mouse) might be limiting?

Reply:
hm yeah that would be nice. Though I'm not sure if DaPars needs a specific nomenclature because the gtf is hardcoded or so. If not, then the best would be to check whether the sample and gtf have same chromosome names. I think that's what we do in the validation within the quantification swf: `summary_workflows/quantification/quantification_dockers/q_validation/validation.py`

Reply:
Oh good point, thanks guys! We need at least one row with leading chr otherwise dapars will error out in the code with a not so obvious error that's it's caused by no leading chr, and that's why we added this check to give a straightforward error message in case none of the rows in the chromosome column have leading chr. I wasn't sure how other non human organisms chromosome nomenclature looks like so I'll change the check to make sure there is at least one row with chr since that's what dapars needs to run without a cryptic error

Reply:
I'm rerunning dapars with the new check I mentioned above, I'll update this PR once it passes the run just to double check! @dominikburri @mrgazzara